consumer/push_consumer.go

```diff
@@ -260,6 +260,11 @@ func (pc *pushConsumer) Subscribe(topic string, selector MessageSelector,
 		return errors2.ErrStartTopic
 	}
 
+	if _, ok := pc.crCh[topic]; !ok {
+		defaultOpts := defaultPushConsumerOptions()
```

Comment:
User may set the value of this parameter, so default options cannot be used here.

Reply:
fixed. i use pc.defaultConsumer.option.ConsumeGoroutineNums